TROUBLESHOOTING.md

```diff
@@ -132,3 +132,21 @@ to be blocked until they are approved (or rejected) by an authorized user. If yo
 may need to ask that the Organization has `quarantine_content_library_items` disabled, or that someone approves or denies the
 upload when the Terraform script is applied.
 
+### Configuring external entity connection (vCenter server, NSX Manager) returns certificate error
```

Comment:
Just had a few thoughts for introduction-like sections which would be useful to add:
- Terraform state command (where .tfstate is saved, how to list it, a link to official docs)
- How to upgrade the provider (version bump and what to expect)

Reply:
Added